src/containers/Home/HomeHero/hello-world.ts

```diff
@@ -12,6 +12,7 @@ const HelloWorld: {[isoLangCode: string]: string} = {
   uk: 'Привіт, світе',
   ur: 'ہیلو ورلڈ',
   vi: 'Xin chào thế giới',
+  nl: 'Hallo Wereld',
```

Comment:
Awesome! I suggest keeping alphabetic order though. 😉

Reply:
My bad, thank you!😊